sigstore/verify/verifier.py

```diff
@@ -113,11 +118,10 @@ def __init__(
         establishing the trust chain for the signing certificate and signature.
         """
         self._rekor = rekor
-
-        self._fulcio_certificate_chain: List[X509] = []
-        for parent_cert in fulcio_certificate_chain:
-            parent_cert_ossl = X509.from_cryptography(parent_cert)
-            self._fulcio_certificate_chain.append(parent_cert_ossl)
+        self._fulcio_X509_certificate_chain: List[X509] = [
+            X509.from_cryptography(parent_cert)
+            for parent_cert in fulcio_certificate_chain
+        ]
```

Comment:
Nitpick, IMO no need for a rename: 

```suggestion
        self._fulcio_certificate_chain: List[X509] = [
            X509.from_cryptography(parent_cert)
            for parent_cert in fulcio_certificate_chain
        ]
```